cmd/kuberhealthy/main.go

```diff
@@ -66,6 +70,7 @@ func init() {
 	flaggy.Bool(&enablePodStatusChecks, "", "podStatusChecks", "Set to false to disable pod lifecycle phase checking.")
 	flaggy.Bool(&enableForceMaster, "", "forceMaster", "Set to true to enable local testing, forced master mode.")
 	flaggy.Bool(&enableDebug, "d", "debug", "Set to true to enable debug.")
+	flaggy.StringSlice(&tolerations, "t", "tolerations", "Pass in a comma separated list of taints to tolerate.")
```

Comment:
The input here does not support CSV format.  You specify the flag multiple times for multiple tolerations.
Maybe change the long name to "toleration", too.  That way the command looks like this:
`./kuberhealthy -tolerate taintName -tolerate Taint2`